master/buildbot/reporters/gerrit_verify_status.py

```diff
@@ -135,6 +138,8 @@ def createStatus(self,
         if duration is not None:
             payload['duration'] = duration
 
+        payload = {'verifications': {name: payload}}
```

Comment:
I'm sorry I do not quite understand: you seem to overwrite whatever `payload` has so far...